packages/applications-service-api/src/controllers/submissions.js

```diff
@@ -10,7 +10,11 @@ module.exports = {
 		let submission = await createSubmission(submissionRequestData);
 
 		if (req.file) {
-			submission = await submitFile(submission, req.file);
+			submission = await submitUserUploadedFile(submission, req.file);
+		}
+
+		if (submission.representation) {
+			submission = await submitRepresentationFile(submission);
```

Comment:
We are exposing details of the file that we create in the API response, I think that's fine, but it may go away in the future so probably want to be careful the FE doesn't build functionality against this.   I think it's fine though.

Reply:
Good point, I'll remove `file` from the response (if it's a submission with `representation`).